virttest/qemu_monitor.py

```diff
@@ -1188,25 +1192,36 @@ def set_migrate_capability(self, state, capability):
 
         :param state: Bool value of capability.
         :param capability: capability which need to set.
+        :raise MonitorNotSupportedMigCapError: if the capability is unknown
         """
         cmd = "migrate_set_capability"
         self.verify_supported_cmd(cmd)
         value = "off"
         if state:
             value = "on"
         cmd += " %s %s" % (capability, value)
-        return self.cmd(cmd)
+        result = self.cmd(cmd)
+        if result != "":
+            raise MonitorNotSupportedMigCapError("Failed to set capability"
+                                                 "%s: %s" %
+                                                 (capability, result))
```

Comment:
Well the return type is not changed, only it can raise an exception in case the set-capability fails, which looks like a good idea. Do you know of any migration tests that currently do that on purpose and would be broken by this commit? In positive testing this is only good thing (in negative testing it'd in-deed require change in test code, which might be costly).